llvm/test/Transforms/LowerTypeTests/cfi-annotation.ll

```diff
@@ -0,0 +1,46 @@
+; RUN: opt -passes=lowertypetests %s -o %t.o
+; RUN: llvm-dis %t.o -o - | FileCheck %s --check-prefix=CHECK-foobar
+; CHECK-foobar: {{llvm.global.annotations = .*[foo|bar], .*[foo|bar],}}
+; RUN: llvm-dis %t.o -o - | FileCheck %s --check-prefix=CHECK-cfi
+; CHECK-cfi-NOT: {{llvm.global.annotations = .*cfi.*}}
+
+target triple = "aarch64-none-linux-gnu"
```

Comment:
If the triple is required, you should add `; REQUIRES: aarch64-registered-target` to the test, otherwise it will probably fail on buildbots without aarch64.